docs/cue/reference/components/sources/internal_metrics.cue

```diff
@@ -932,6 +984,16 @@ components: sources: internal_metrics: {
 			required:    true
 			examples: [_values.local_host]
 		}
+		_instance: {
+			description: "The Vector instance identified by host and port."
+			required:    true
+			examples: [_values.instance]
+		}
+		_job: {
+			description: "The name of the job producing Vector metrics."
+			required:    true
+			default:     "vector"
+		}
```

Comment:
I don't think this is a tag we add?

Reply:
Ah, yeah, that snuck back in there. Thanks for catching.